src/components/AmountEditable.tsx

```diff
@@ -229,12 +234,53 @@ export const AmountEditable: ParentComponent<{
                     onFocus={() => focus()}
                 />
             </div>
-            <Show when={props.maxAmountSats}>
-                <p class="flex gap-2 px-4 py-2 text-sm font-light text-m-grey-400 md:text-base">
-                    {`${i18n.t("receive.amount_editable.balance")} `}
-                    <AmountSmall amountSats={props.maxAmountSats!} />
-                </p>
+            <Show when={props.methods?.length && props.activeMethod}>
+                <MethodChooser
+                    methods={props.methods!}
+                    activeMethod={props.activeMethod!}
+                    setChosenMethod={props.setChosenMethod}
+                />
             </Show>
         </div>
     );
 };
+
+function MethodChooser(props: {
+    activeMethod: MethodChoice;
+    methods: MethodChoice[];
+    setChosenMethod?: (method: MethodChoice) => void;
+}) {
+    function setNextMethod() {
+        const activeIndex = props.methods.findIndex(
+            (m) => m.method === props.activeMethod.method
+        );
+        const nextMethod =
+            props.methods[
+                activeIndex === props.methods.length - 1 ? 0 : activeIndex + 1
+            ];
+        props.setChosenMethod && props.setChosenMethod(nextMethod);
+    }
+    return (
+        <button
+            onClick={setNextMethod}
+            disabled={props.methods.length === 1}
+            class="flex gap-2 rounded px-2 py-1 text-sm font-light text-m-grey-400 md:text-base"
+            classList={{
+                "border-b border-t border-b-white/10 border-t-white/50 bg-neutral-700":
+                    props.methods?.length > 1
+            }}
+        >
+            <AmountSats
+                amountSats={props.activeMethod.maxAmountSats!}
+                denominationSize="sm"
+                icon={
+                    props.activeMethod.method === "lightning"
+                        ? "lightning"
+                        : props.activeMethod.method === "onchain"
+                        ? "chain"
+                        : "community"
```

Comment:
do we want this commutiny icon here?

Reply:
might be better to have this just as a function next to the definition of `method` that coverts to an icon so we don't have to remember to change this too when we add another method